debian/control

```diff
@@ -9,8 +9,8 @@ Build-Depends: bash-completion,
                fakeroot,
                gettext,
                gnupg2,
+               golang (>=2:1.6) | golang-1.6,
                golang-check.v1-dev,
-               golang-go,
```

Comment:
You may as well just depend on golang-1.6-go unconditionally, at least until we start thinking about demoting that from main in 17.04.